Header header
float64 tip_pressure
bool is_touching